exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/PlannerSettings.java

```diff
@@ -212,6 +219,14 @@ public static long getInitialPlanningMemorySize() {
     return INITIAL_OFF_HEAP_ALLOCATION_IN_BYTES;
   }
 
+  public double getFilterMinSelectivityEstimateFactor() {
+    return options.getOption(FILTER_MIN_SELECTIVITY_ESTIMATE_FACTOR.getOptionName()).float_val;
```

Comment:
Make the option validators above typed:
`public static final FloatValidator FILTER_MIN_SELECTIVITY_ESTIMATE_FACTOR ...`

and change this line to:
`return options.getOption(FILTER_MIN_SELECTIVITY_ESTIMATE_FACTOR);`

Same for the other option.